internal/helper.go

```diff
@@ -0,0 +1,37 @@
+package internal
+
+import (
+	"io/fs"
+	"path/filepath"
+	"strings"
+)
+
+func PathsFor(wd string, filesNames ...string) []string {
```

Comment:
```suggestion
func PathsFor(wd string, fileNames ...string) []string {
```

Reply:
yes that function had 2 other places where that typo was used, just changed them, And global searched `filesNames` and no other match.